core/deployment/src/main/java/io/quarkus/deployment/pkg/NativeConfig.java

```diff
@@ -77,13 +79,27 @@
 
     /**
      * Defines the user country used for building the native executable.
+     * It also serves as te default Locale country for the native executable application runtime.
+     * e.g. US or FR as defined by ISO 3166-1 alpha-2 codes.
      * <p>
      * Defaults to the system one.
      */
     @ConfigItem(defaultValue = "${user.country:}")
     @ConvertWith(TrimmedStringConverter.class)
     public Optional<String> userCountry;
 
+    /**
+     * Defines the locales to be accessible in the native executable.
+     * The format is a comma separated list of IETF BCP 47 language tags,
+     * e.g. "fr,de,cs" or "de-AT,ja-JP" etc.
+     * See also LocalesBuildTimeConfig for a separate, native-image unrelated usage.
+     * <p>
+     * Defaults to empty, i.e. not explicitly set during native-image build.
+     */
+    @ConfigItem
+    @ConvertWith(TrimmedStringConverter.class)
+    public Optional<String> locales;
```

Comment:
Should we introduce specific locales for native while we have this one already: https://github.com/quarkusio/quarkus/blob/main/core/runtime/src/main/java/io/quarkus/runtime/LocalesBuildTimeConfig.java (and it needs to be set properly if you want the HV locale support work properly).

My guess is that we could also deprecate the two others that are already there (but still support them for now) and use `quarkus.default-locale` instead.

Reply:
Ack. Done.